SPECS/ibarr/ibarr.spec

```diff
@@ -55,6 +54,8 @@ rm -rf $RPM_BUILD_ROOT
 /lib/systemd/system/%{name}.service
 
 %changelog
+* Mon Sep 15 2025 Elaheh Dehghani <edehghani@microsoft.com>
```

Comment:
Could I ask you to also include the version and release number in the changelog like we do in other specs. I think these few added by Binu are missing them, which is unfortunate, but this helps to keep track of the changes.